ir/helper.go

```diff
@@ -220,15 +220,6 @@ type LocalIdent struct {
 	LocalID   int64
 }
 
-// NewLocalIdent returns a new local identifier based on the given string. An
```

Comment:
@dannypsnl After looking a bit more carefully. Can we really remove this function? Since it was exported others may depend on it. The API of `llir/llvm` is still in `v0.x` so it has not yet reached version 1.0, but we can still try to strive for maintaining API stability, or give alternatives to transition to when changing the API.

Reply:
I would check it later.

Reply:
Well, I think users would use Instruction to produce identifier, rather than using this API. I think API stability is good, but only when the API is easy-correct API. However, I don't think `NewLocalIdent` is. I still would like to remove it. I think we can make an overview and decide which API should be moved to `internal` package.